config/webpack/loaders.js

```diff
@@ -67,11 +67,12 @@ module.exports = [
       {
         loader: 'sass-loader',
         options: {
+          sourceMap: true,
           prependData: () => {
             return `$img-base-path: '${appBasePath}';`;// Path variable for login and about modal images.
           },
           sassOptions: {
-            sourceMap: true,
+            outputStyle: "compressed",
```

Comment:
please use this in single quotes to avoid lint error
`outputStyle: 'compressed',`

Reply:
fixed